rest-api-spec/src/main/resources/rest-api-spec/api/cluster.health.json

```diff
@@ -12,6 +12,12 @@
         }
       },
       "params": {
+        "expand_wildcards": {
```

Comment:
The REST API supports other indices options like `ignore_unavailable` but those options are overridden in the `TransportClusterHealthAction` to report a RED status is an index is missing (or just ignore it). 

This is why this sole option is documented.